### ISSUE TITLE
Renamed app using the original name in the home screen and some suggestions.

### ISSUE THREAD

**Issue by Helfexen** [jooooscha/Olauncher#19]
**Bugs/Glitches:**
- You can see hidden apps when you are editing home screen buttons.
- When you add a renamed app to the home screen it is using the original name instead.

**QoL Suggestions:**
- Expose hidden apps in the Olauncher's menu so people can unhide when necessary.
- A new gesture (maybe pinch-in, pinch-out?) for recent apps screen so we can close things.
- Creating categories in the app drawer for better organization.

Already installed on three devices, it's the first time for me to use a launcher without tweaking for hours.

**Comment by jooooscha** [comment]
Thank you for your suggestions. Here are my thoughts an this.

* Your first point does not seem to be a bug. For example, one might want to hide an app because they want to place that app one the home screen.
* This is, indeed, not optimal but the way the original author designed the app it would be unfeasible to implement, and as you can change the app name on the home screen I would give this an immediate priority
* I am not sure what you mean by this. Which menu do you mean? Further, you can already unhide apps by clicking on the app name in the settings. This is original behavior, but as I see it it is not intuitive. I will look into changing this.
* Again, I am not sure what you mean. Where is the recent apps screen? What do you want to close?
* This could be a question of debate. I would argue the purpose of this launcher design is to search for apps by text. I don't see how categories would help this design without adding too much clutter.

To summarize, I will look into improving the unhide experience, but would need more details on the other points.

**Comment by jooooscha** [comment]
Yes, you can actually change the app name on the home screen. I agree that it is unintuitive, but its unchanged from the original version. 
To change an app on the home screen you just long-press on it again and type your name. A rename button will appear.

Removing One UI is not an official feature supported by this app. Also, this app does not have any access to the recently used apps screen; this is an android system feature.

**Comment by jooooscha** [comment]
I have made the hidden apps list more visible in #22.

As far as I can see are the other issues solved, as well. Closing this
